blockchain/types/block.go

```diff
@@ -64,10 +64,12 @@ type Header struct {
 	GasUsed     uint64         `json:"gasUsed"          gencodec:"required"`
 	Time        *big.Int       `json:"timestamp"        gencodec:"required"`
 	// TimeFoS represents a fraction of a second since `Time`.
-	TimeFoS    uint8  `json:"timestampFoS"     gencodec:"required"`
-	Extra      []byte `json:"extraData"        gencodec:"required"`
-	Governance []byte `json:"governanceData"        gencodec:"required"`
+	TimeFoS    uint8  `json:"timestampFoS"              gencodec:"required"`
+	Extra      []byte `json:"extraData"                 gencodec:"required"`
+	Governance []byte `json:"governanceData"            gencodec:"required"`
 	Vote       []byte `json:"voteData,omitempty"`
+
+	BaseFee *big.Int `json:"baseFeePerGas"    rlp:"optional"`
```

Comment:
Doesn't it need `omitempty` ?

Reply:
Yes it doesn't. It is used for RPC encoding/decoding. I think `omitempty` is used when the field take a big size or need logic checking whether there is the field or not.

Reply:
When a user tries to marshal/unmarshal old block headers that don't have `BaseFee` field, he could experience unexpected errors without `omitempty`.

Reply:
Do you know elsewhere the header is marshaled/unmarshaled? I found It only in `api_ethereum.go` / `node/cn/filters/api.go` and there have been `baseFeePerGas` field already. I don't think it cause an unexpected errors.

Reply:
A Klaytn node may not marshal/unmarshal with Marshal/Unmarshal methods. But, our code can be used as a part of client application since this project provides golang client code also. A client user could try to unmarshal `getHeader~` API returns with this struct.

Reply:
@aidan-kwon `omitempty` tag was added.